docs/en/latest/how-to-build.md

```diff
@@ -94,6 +95,23 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
   make install
   ```
 
+## Step 2: Install ETCD
+
+This step is required when installing Apache APISIX via RPM or Docker.
```

Comment:
Maybe we have an independent doc for "install from source"?

Reply:
I didn't find this document, I've changed the expression.